examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -267,11 +267,13 @@ export class TaskList extends DataObject implements ITaskList {
         // sync'ing perhaps this should only include ack'd changes (by spinning up a second local client same
         // as what we do for summarization).
         const tasks = this.getTasks();
-        const taskStrings = tasks.map((task) => {
-            return `${task.id}:${task.name.getText()}:${task.priority.toString()}`;
-        });
-        const stringDataToWrite = `${taskStrings.join("\n")}`;
-
+        const formattedTasks = {}
+        tasks.map((task) => {
```

Comment:
Since we're not assigning the results of `map` to a variable, might be better to use `foreach` instead.